internal/version/version.go

```diff
@@ -0,0 +1,10 @@
+package version
+
+var (
+	// dev is the fallback version, as the variable is set to the correct version by the Dockerfile during build time.
+	version string = "dev"
```

Comment:
When building with docker-build locally and applying it to the cluster, the version is actually empty, not `dev`.